pennylane/qnode.py

```diff
@@ -1041,7 +1041,7 @@ def _pd_analytic_var(self, param_values, param_idx, **kwargs):
                 # the square of the observable
                 A = np.kron(A, A.T)
 
-                new = qml.expval(qml.ops.PolyXP(A, w, do_queue=False))
+                new = qml.expval(qml.ops.PolyXP(A, w))
```

Comment:
Just trying to follow along, but since we are _inside_ a QNode here, is `_current_context` defined when these operations are instantiated? If so, the behaviour might change, as this will cause queuing to take place.

But from looking at the code I am unsure...

Reply:
I don't think so. You have to be inside a `QNode` context, not necessarily inside a `QNode` class

Reply:
I had a look at the qnode.py file, and the current context is only set for a few lines during qfunc execution. And then remains unset during evaluation of other methods. So should be all good.